Test Failure: io.openliberty.microprofile.telemetry.internal.tests.JaegerOltpTest.testBasic

RTC 294532

```
testBasic:java.lang.Throwable: 2023-01-25-04:10:50:268 UNAVAILABLE: io exception
    at componenttest.custom.junit.runner.FATRunner.newThrowableWithTimeStamp(FATRunner.java:331)
    at componenttest.custom.junit.runner.FATRunner.access$700(FATRunner.java:69)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:303)
    at org.testcontainers.containers.FailureDetectingExternalResource$1.evaluate(FailureDetectingExternalResource.java:29)
    at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:356)
    at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:177)
Caused by: io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
    at io.grpc.StatusRuntimeException.fillInStackTrace(StatusRuntimeException.java:68)
    at io.grpc.StatusRuntimeException.<init>(StatusRuntimeException.java:58)
    at io.grpc.StatusRuntimeException.<init>(StatusRuntimeException.java:50)
    at io.grpc.Status.asRuntimeException(Status.java:535)
    at io.grpc.stub.ClientCalls$BlockingResponseStream.hasNext(ClientCalls.java:649)
    at io.openliberty.microprofile.telemetry.internal.utils.jaeger.JaegerQueryClient.consumeChunkedResult(JaegerQueryClient.java:252)
    at io.openliberty.microprofile.telemetry.internal.utils.jaeger.JaegerQueryClient.getSpansForTraceId(JaegerQueryClient.java:105)
    at io.openliberty.microprofile.telemetry.internal.utils.jaeger.JaegerQueryClient.waitForSpansForTraceId(JaegerQueryClient.java:165)
    at io.openliberty.microprofile.telemetry.internal.tests.JaegerBaseTest.testBasic(JaegerBaseTest.java:69)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:203)
Caused by: java.io.IOException: Operation timed out
    at java.base/sun.nio.ch.FileDispatcherImpl.read0(Native Method)
    at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
    at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
    at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
    at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
    at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:356)
    at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:259)
    at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
    at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
    at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
    at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
    at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
    at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
    at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
    at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
    at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
    at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
    at java.base/java.lang.Thread.run(Thread.java:866)
```

--- a/dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/bnd.bnd
+++ b/dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/bnd.bnd
@@ -22,7 +22,7 @@ fat.project: true
 
 #Any changes to these names must be copied to TestConstants
 fat.test.container.images: \
-    jaegertracing/all-in-one:1.39, \
+    jaegertracing/all-in-one:1.54, \
     openzipkin/zipkin-slim:2.23, \
     otel/opentelemetry-collector:0.74.0 ,\
     alpine:3.16

--- a/dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/TestConstants.java
+++ b/dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/TestConstants.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2022, 2024 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
@@ -83,7 +83,7 @@ public class TestConstants {
 
     //Docker image names. These must be kept in sync with bnd.bnd
     public static final DockerImageName DOCKER_IMAGE_OPENTELEMETRY_COLLECTOR = DockerImageName.parse("otel/opentelemetry-collector:0.74.0");
-    public static final DockerImageName DOCKER_IMAGE_ALL_IN_ONE = DockerImageName.parse("jaegertracing/all-in-one:1.39");
+    public static final DockerImageName DOCKER_IMAGE_ALL_IN_ONE = DockerImageName.parse("jaegertracing/all-in-one:1.54");
     public static final DockerImageName DOCKER_IMAGE_ZIPKIN_SLIM = DockerImageName.parse("openzipkin/zipkin-slim:2.23");
 
 

--- a/dev/io.openliberty.org.testcontainers/cache/images
+++ b/dev/io.openliberty.org.testcontainers/cache/images
@@ -7,7 +7,7 @@ gvenzl/oracle-xe:21.3.0-full-faststart
 gvenzl/oracle-xe:21.3.0-slim-faststart
 infinispan/server:10.0.1.Final
 infinispan/server:13.0.10.Final
-jaegertracing/all-in-one:1.39
+jaegertracing/all-in-one:1.54
 jboss/keycloak:16.1.1
 jiwoo/simple-keyserver:1.0
 jonhawkes/postgresql-ssl:1.0

